Sources/XiEditor/RPCSending.swift

```diff
@@ -19,6 +19,10 @@ import AppKit
 /// These logs can be used for profiling & debugging.
 let XI_RPC_LOG = "XI_CLIENT_RPC_LOG"
 
+let NEW_LINE = [0x0a as UInt8]
+let SEND_LOG_PREFIX = "SEND:".data(using: .utf8)!
+let RECV_LOG_PREFIX = "RECV:".data(using: .utf8)!
```

Comment:
Perhaps use `[SEND]` and `[RECV]`?

Reply:
I have been debating formatting.  If I do, then each message now takes more than one line per message, which I think might make it a bit harder to scan and find the message you're after.

For example:

```
SEND: {
  "params":{
    "enabled":true
  }, 
  "method":"tracing_config"
}
SEND:{
  "params":{
    "client_extras_dir":"...",
    "config_dir":"..."
  },
  "method":"client_started"
}
RECV:{
  "method":"available_languages",
  "params":{
    "languages":[
      "ASP","ActionScript","AppleScript","Batch File","BibTeX","Bourne Again Shell (bash)","C","C#","C++","CSS","Clojure","D","Diff","Erlang","Git Attributes","Git Commit","Git Config","Git Ignore","Git Link","Git Log","Git Rebase Todo","Go","Graphviz (DOT)","Groovy","HTML","HTML (ASP)","HTML (Erlang)","HTML (Rails)","HTML (Tcl)","Haskell","JSON","Java","Java Properties","Java Server Page (JSP)","JavaScript","JavaScript (Rails)","LaTeX","Lisp","Literate Haskell","Lua","MATLAB","Makefile","Markdown","NAnt Build File","OCaml","OCamllex","OCamlyacc","Objective-C","Objective-C++","PHP","Pascal","Perl","Plain Text","Python","R","Rd (R Documentation)","Regular Expression","Ruby","Ruby Haml","Ruby on Rails","Rust","SQL","SQL (Rails)","Scala","TOML","Tcl","TeX","Textile","XML","YAML","reStructuredText"
    ]
  }
}
```

I could be convinced to apply formatting if folks think it'd be useful. :) 

I do like you're idea of naming though.. I could easily use `CLIENT` and `CORE` and still retain the "column" layout. I'll update the PR with that change tonight.

```
[CLIENT] {"method":"new_view","id":0,"params":{}}
[CORE]   {"id":0,"result":"view-id-2"}
```